artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/converter/AMQPMessageSupport.java

```diff
@@ -198,6 +200,8 @@
    public static final Symbol JMS_DEST_TYPE_MSG_ANNOTATION = getSymbol("x-opt-jms-dest");
    public static final Symbol JMS_REPLY_TO_TYPE_MSG_ANNOTATION = getSymbol("x-opt-jms-reply-to");
 
+   public static final Symbol INGRESS_TIME = getSymbol(HDR_INGRESS_TIME);
```

Comment:
Might be good to group and align the name with the other _MSG_ANNOTATION constants above?

Reply:
I had it that way at first, but I was getting an "Illegal forward reference" error due to the location of `HDR_INGRESS_TIME`. Moving `HDR_INGRESS_TIME` would also require moving `X_OPT_PREFIX` so it seemed simpler to just put `INGRESS_TIME` where it is.

Reply:
I wasnt suggesting anything that should cause illegal forward references. I simply mean delete the empty line above it, and add _MSG_ANNOTATION to the name like the other message annotation constants that exist.

Reply:
Ok I see now what you thought I was saying - I didt mean the internal property names ~25 lines up used to carry detail during protocol conversions, just the other specific message annotation symbols a line up.

Reply:
Done.